balena/weston-init

```diff
@@ -13,6 +13,9 @@ cat << IEOF > /weston.ini
 [core]
 require-input=false
 
+[output]
+name=DSI-1
```

Comment:
is this necessary?

Reply:
(because this will have an unintended effect - if no resolution cap is in place and/or the max res meets the cap, no other output sections will be created, which will theoretically disable HDMI out)

Reply:
no. definitely not. It was a mistake. Thanks for pointing that.